tests/requirements-libraries.txt

```diff
@@ -26,7 +26,8 @@ sphinx ; python_version != '3.3'
 
 pyzmq
 zope.interface  # Required for test_namespace_package
-numpy
+numpy ; python_version < '3.0' or python_version >= '3.4'
+numpy<1.12 ; python_version >= '3.0' and python_version < '3.4'
```

Comment:
Did numpy 1.12 end support for Python 3.4, while keeping support for 2.7? This is a curious! Can you please add a comment line why this is necessary. Thanks.